src/pages/studyView/addChartButton/customCaseSelection/CustomCaseSelection.tsx

```diff
@@ -73,14 +73,27 @@ export default class CustomCaseSelection extends React.Component<ICustomCaseSele
 
     @autobind
     @action
-    onClick() {
-        let cases = this.props.selectedSamples.map(sample => {
+    onClick(event: any) {
+      let cases;
+      let caseInverted;
+        if(event.target.id == "selected") {
+          cases = this.props.selectedSamples.map(sample => {
+              return `${sample.studyId}:${(this.caseIdsMode === ClinicalDataTypeEnum.SAMPLE) ? sample.sampleId : sample.patientId}${this.props.disableGrouping ? '' : ` ${DEFAULT_GROUP_NAME_WITHOUT_USER_INPUT}`}`
+          });
+          if (this.caseIdsMode === ClinicalDataTypeEnum.PATIENT) {
+              cases = _.uniq(cases);
+          }
+          this.content = cases.join("\n")
+        }
+        if(event.target.id == "unselected") {
+          caseInverted = this.props.allSamples.map(sample => {
```

Comment:
When you choose `unselected`, it's the complement set of `this.props.selectedSamples` in `this.props.allSamples`